cmd/security-agent/subcommands/workloadlist/command_test.go

```diff
@@ -0,0 +1,22 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package workloadlist
+
+import (
+	"testing"
+
+	"github.com/DataDog/datadog-agent/cmd/security-agent/command"
+	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+)
+
+func TestWorkloadListCommand(t *testing.T) {
```

Comment:
Should we add test that validate the URL that gets build base on the CLI params?